app/src/main/java/com/github/h3lp3rs/h3lp/professional/VerificationActivity.kt

```diff
@@ -0,0 +1,140 @@
+package com.github.h3lp3rs.h3lp.professional
+
+import android.app.Activity
+import android.content.Intent
+import android.net.Uri
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import android.util.Log
+import android.webkit.MimeTypeMap
+import android.widget.*
+import androidx.activity.result.contract.ActivityResultContracts
+import com.github.h3lp3rs.h3lp.R
+import com.github.h3lp3rs.h3lp.database.Databases
+import com.github.h3lp3rs.h3lp.signin.GoogleSignInAdapter
+import com.github.h3lp3rs.h3lp.signin.SignInActivity
+import com.google.firebase.storage.FirebaseStorage
+import com.google.firebase.storage.StorageReference
+import com.squareup.picasso.Picasso
+
+/**
+ * Activity to verify the professional status of the user before redirecting to the professional portal
+ */
+class VerificationActivity : AppCompatActivity() {
+
+    // Some attributes are vars only for testing purposes
+    companion object {
+        // Uri of the image file = proof of status
+        var imgUri: Uri? = null
+
+        // Database representing the professional users authenticated
+        val db = Databases.databaseOf(Databases.PRO_USERS)
+
+        // Reference to the Firebase cloud storage
+        var storageRef = FirebaseStorage.getInstance().getReference("uploads")
```

Comment:
We could have something like the SignIn class to avoid having to use a public var here (it'd also allow us to change our "uploads" database at will)

Reply:
Good point! I should have thought of doing it this way 👍🏻